kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/servlet/LogServlet.java

```diff
@@ -63,6 +68,17 @@ protected void doGet(HttpServletRequest httpServletRequest, HttpServletResponse
             return;
         }
 
+        PrivilegedExecutorService pes = null;
```

Comment:
Can't the pes be obtained in the write method, since it seems the only usage point?

Reply:
@MMaiero Yes, but then I've to pass to the `writeJournaldLog` also the locator and the session...